README.md

```diff
@@ -1,15 +1,107 @@
+<div align="center">
+
+# 🖌 vello
+
+**An experimental GPU compute-centric 2D renderer**
+
+[![Xi Zulip](https://img.shields.io/badge/Zulip-Chat-blue?logo=Zulip)](https://xi.zulipchat.com/)
+[![dependency status](https://deps.rs/repo/github/linebender/vello/status.svg)](https://deps.rs/repo/github/linebender/vello)
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](#License)
+<!-- [![Crates.io](https://img.shields.io/crates/v/vello.svg)](https://crates.io/crates/vello) -->
+<!-- [![Docs](https://docs.rs/vello/badge.svg)](https://docs.rs/vello) -->
+<!-- [![Build status](https://github.com/linebender/vello/workflows/CI/badge.svg)](https://github.com/linebender/vello/actions) -->
+
+</div>
+
+Vello is a 2d graphics middleware for [`wgpu`]. 
+<!-- It allows effeciently drawing large numbers of 2d elements with interactive performance. TODO: Verify this claim -->
+
+<!-- Impressive picture here -->
+
+It is used as the rendering backend for [xilem], a UI toolkit.
+
+## Examples
+
+Our examples are provided in seperate packages in the [`examples`](examples) folder. 
+This allows them to have independent dependencies and faster builds.
+Examples must be selected using the `--package` (or `-p`) Cargo flag.
+
+### Winit
+
+Our [winit] example ([examples/with_winit](examples/with_winit)) demonstrates rendering to a [winit] window.
+It also includes a collection of test scenes showing the capabilities of vello.
+One of these scenes uses an incomplete svg parser/renderer to render the [GhostScript tiger].
+
+```shell
+cargo run -p with_winit
+```
+
+### Web
+
+Because we use compute shaders, we cannot run on the [`wgpu`] webgl2 backend. A simplified version of the pipeline ran 
```

Comment:
```suggestion
Because Vello relies heavily on compute shaders, we rely on the emerging WebGPU standard to run on the web. Until browser support becomes widespread, it will probably be necessary to use development versions (Chrome Canary) and explicitly enable WebGPU. The following command builds a web version of the demo.
```